### ISSUE TITLE
Thrown errors should be `Error` instances.

### ISSUE THREAD

**Issue by bendavis78** [molleindustria/p5.play#203]
For example: 
```javascript
 throw('addImage error: allowed usages are <image> or <label>, <image>');
```
should be:
```javascript
 throw(new Error('addImage error: allowed usages are <image> or <label>, <image>'));
```

The reason for this is that a stack trace is not included when you throw a string. This makes things difficult to debug. There's no way to know what line this error occurred on.

**Comment by quinton-ashley** [comment]
I will do this fix!

**Comment by GoToLoop** [comment]
Just a tiny nitpick: `throw` is a JS keyword command which doesn't need any parens: :nerd_face:
https://developer.Mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw

**Comment by quinton-ashley** [comment]
@GoToLoop Yup true. Fixed in version 2.0.1
